app/controllers/admin/users_controller.rb

```diff
@@ -2,6 +2,9 @@ class Admin::UsersController < Admin::BaseController
 
   def show
     @user = User.find(params[:id])
+    if @user.role == 'merchant'
```

Comment:
We could use the enum helper:

`if @user.merchant?`

Reply:
Just implemented this and we should be good!